index.js

```diff
@@ -1,16 +1,14 @@
 const VueAutoRoutingPlugin = require('vue-auto-routing/lib/webpack-plugin')
 
-module.exports = (api, options) => {
-  const opts =
-    (options.pluginOptions && options.pluginOptions.autoRouting) || {}
+const defaultOptions = {
+  pages: 'src/pages',
+  nested: true,
+}
 
+module.exports = (api, options) => {
   const pluginOptions = {
-    pages: 'src/pages',
-    nested: true
-  }
-
-  if (opts.chunkNamePrefix != null) {
-    pluginOptions.chunkNamePrefix = opts.chunkNamePrefix
+    ...defaultOptions,
+    ...(options.pluginOptions && options.pluginOptions.autoRouting),
```

Comment:
```suggestion
    ...((options.pluginOptions && options.pluginOptions.autoRouting) || {}),
```

Reply:
Not sure what does this solve? `undefined` is a valid argument for the object rest operator.

If it were the Typescript code, it could be even more concise with `...options.pluginOptions?.autoRouting`.

Reply:
Oh, I misunderstood the behavior! Let's go with the current code, then.